daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/FrontStack.scala

```diff
@@ -109,6 +109,8 @@ final class FrontStack[+A] private (fq: FQ[A], len: Int) {
     case _ => false
   }
 
+  override def hashCode(): Int = toImmArray.hashCode()
+
```

Comment:
missing the same thing for BackStack

Reply:
We need a tests for that as well.

Reply:
Yes, there are no tests at all yet for FrontStack and BackStack, at least as far as I can tell.

Reply:
@remyhaemmerle-da: if you want to add tests, please feel free to take over the PR.